tests/commands.test.ts

```diff
@@ -0,0 +1,59 @@
+import { addNewClass } from '../src/commands'
+import  { fileExists, writeToFile } from '../src/utils/file'
+import { getProjectConfiguration } from '../src/utils/configuration'
+import { JsRuntime, Language, ProjectConfiguration, SdkConfiguration } from '../src/models/projectConfiguration'
+
+jest.mock('../src/utils/file')
+jest.mock('../src/utils/configuration')
+
+beforeEach(() => {
+    jest.clearAllMocks()
+})
+
+describe('addNewClass', () => {
+    test('throws error if incompatible extension type', async () => {
```

Comment:
`extension type` is confusing. 

I would change the message to: `throws error if class type is not supported`